### PR TITLE
Container

### PR DESCRIPTION
**Pull request by Firtzberg**
It is my honor and great pleasure to have been assigned the first ticket ever in the SERCOMP project. XD
[SERCOMP-1](https://55places.atlassian.net/browse/SERCOMP-1)
Implements https://github.com/neighborhoods/KojoWorkerDecoratorComponent/issues/5.
This PR introduces breaking changes, since a fabricated class implementation changed. Because of this it is desired to have https://github.com/neighborhoods/KojoWorkerDecoratorComponent/pull/9 sorted out first, released as version 1.1 and then release these changes as version 2.0.

Is it possible to somehow fabricate a test?

### CODE REVIEW

**Comment by Firtzberg** [comment]
TODO: Add upgrade guide after it is decided what will be included in version 1 and what goes into version 2.

File: README.md [hunk 1]
```diff
@@ -206,6 +206,7 @@ $containerBuilder->getDiscoverableDirectories()->addDirectoryPathFilter(
     '../vendor/neighborhoods/kojo-worker-decorator-component/src'
 );
 ```
+The container building and caching be extracted from the Proxy into a dedicated class in order to test it.
```
Comment (Firtzberg) [inline README.md]: *can* be

**Review by arielallon — APPROVED** [review]
this is also a breaking change.
we should release all of this as part of v2
and have an upgrade guide

File: README.md (hunk 1 above)
Comment (Firtzberg) [reply README.md]: solved

**Pull request merged**